bundles/org.openhab.binding.kvv/src/main/java/org/openhab/binding/kvv/internal/KVVStationConfig.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.kvv.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * Holds information parameters.
+ *
+ * @author Maximilian Hess - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class KVVStationConfig {
+
+    /** the id of the station */
+    public @Nullable String stationId;
+
+    /** maximum number of traines being queried */
+    public int maxTrains;
+
+    /** the update interval */
+    public int updateInterval;
+
+    /** API key of the KVV API */
+    public @Nullable String apiKey;
```

Comment:
I would say that this is a parameter for the bridge otherwise your bridge does not make much sense if you have to respecify the api-key every time. I suppose the same holds for `maxTrains` and `updateInterval`. So I would expect to configs on with global settings for the bridge and one with a station-id for station-things.